package.json

```diff
@@ -34,9 +34,12 @@
   },
   "dependencies": {
     "@bigtest/interactor": "^0.28.2",
+    "@date-io/date-fns": "^1.3.13",
     "@material-ui/core": "^4.11.3",
     "@material-ui/icons": "^4.11.2",
+    "@material-ui/pickers": "^3.3.10",
     "@material-ui/styles": "^4.11.3",
```

Comment:
Are these dependencies or dev dependencies?

Reply:
Should we clean up deps in this PR?

Reply:
Good point. I've create a ticket for this here: https://github.com/thefrontside/material-ui-interactors/issues/13